lstchain/reco/reconstructor.py

```diff
@@ -24,6 +25,16 @@ class TimeWaveformFitter(TelescopeComponent):
                                       allow_none=False).tag(config=True)
     crosstalk = FloatTelescopeParameter(default_value=0, help='Average pixel crosstalk.',
                                         allow_none=False).tag(config=True)
+    spatial_selection = Unicode(default_value='dvr',
+                                help='Method to select pixels to perform the likelihood fit. Can be:\n'
+                                     'hillas : use the hillas length and width times sigma_space as an ellipsis.'
+                                     '\ndvr : use data volume reduction logic with dvr_pic_threshold and'
+                                     ' dvr_pix_for_full_image',
+                                allow_none=False).tag(config=True)
+    dvr_pic_threshold = Int(8, help='Pixel charge threshold for dvr like pixel selection.',
```

Comment:
Just for code consistency, can you add `default_value=` to justify the first argument for this and some other parameters?